adserver/templates/adserver/email/flight_wrapup.html

```diff
@@ -11,8 +11,8 @@
 <ul>
   {% for flight in completed_flights %}
   <li>
-    <a href="{{ site.domain }}{{ flight.get_absolute_url }}">{{ flight.name }}</a>
-    <span> (<abbr title="{% trans 'Click-through rate' %}">{% trans 'CTR' %}</abbr>: {{ flight.ctr }}%)</span>
+    <a href="{{ site.domain }}{{ flight.get_absolute_url }}?#ref=email-wrapup">{{ flight.name }}</a>
```

Comment:
Isn't a `#` in a URL going to be interpreted as a fragment? Think we just want `?ref=`

Reply:
You are correct